airflow/providers/amazon/aws/operators/redshift_sql.py

```diff
@@ -44,7 +44,8 @@ class RedshiftSQLOperator(BaseOperator):
 
     template_fields: Sequence[str] = ('sql',)
     template_ext: Sequence[str] = ('.sql',)
-    template_fields_renderers = {"sql": "postgresql"}
+    # TODO: Update ``sql`` renderer to "postgresql" when the lexer is available in Airflow 2.3.0.
```

Comment:
I  think it would be better to dynamically check the renderers available in this case. Otherwise, even if we release 2.3.0, we will not be able to change it as the providers are 2.1.0+

Reply:
Ah you're right. I'll update.